src/stylesheets/colors.scss

```diff
@@ -0,0 +1,23 @@
+$green-tint: #F3F7E6;
+$green-bright: #008918;
+$medium-gray: #DDD;
+
+// Some NYPL specific colors: https://nypl.github.io/design-toolkit/sections/color.html.
+$nypl-red: #D0343A;
```

Comment:
We are using NYPL specific colors but for circulation manager we didn't use the `$nypl-` prefix because the project can be run by anyone. The same idea applies here since NYPL and DPLA (and anyone else) can use this registry admin.